week-2-practise/level-3/3-sort.js

```diff
@@ -1,10 +1,17 @@
 // 1. Update the variable `sortedNums`.
 // It should contain the values of `nums` and `nums2` in ascending order
 // Tip: you might need to read the documentation for .sort (search "mdn array sort")
-
+function comapre(element1, element2) {
+  return element1 - element2;
```

Comment:
I'm slightly amazed this works! In a compare function though you should try using < and > instead.
(Also, there's a typo - it should be compare, not comapre).